src/supercell.jl

```diff
@@ -32,9 +32,13 @@ function cell_to_supercell(basis::PlaneWaveBasis)
     supercell_size = basis.kgrid
     supercell = create_supercell(model.lattice, model.atoms, model.positions, supercell_size)
     supercell_fft_size = basis.fft_size .* supercell_size
+    # Keep some values unchanged from the cell model
+    cell_kwargs = (; terms=model.term_types, model.temperature, model.smearing, model.εF,
+                   # Can be safely disabled; this has already been checked for basis.model
+                   disable_electrostatics_check=true)
     # Assemble new model and new basis
     model_supercell = Model(supercell.lattice, supercell.atoms, supercell.positions;
-                            terms=model.term_types, symmetries=false)
+                            cell_kwargs..., symmetries=false)
```

Comment:
There is even more. Just use the pattern of 
https://github.com/JuliaMolSim/DFTK.jl/blob/master/src/postprocess/stresses.jl
Btw there the εF is not passed, maybe one should do a pass to fix the stresses and equivalent places where the model is reconstructed elsewhere (e.g. serialisation, forward-diff are places that come to my mind.